examples/esp32/xively_demo/main/xively_if.c

```diff
@@ -0,0 +1,617 @@
+#include <stdio.h>
```

Comment:
We don't usually call our demo applications "interfaces".  @atigyi what are your thoughts about this?  Should we be more consistent or are self contained examples just that - self contained?

Reply:
How about xively_esp32_if ?